pep-9999.rst

```diff
@@ -0,0 +1,180 @@
+PEP: 9999
+Title: Callable Modules
+Author: Amethyst Reese <amethyst at n7.gg>
+Sponsor: Łukasz Langa <lukasz at python.org>
```

Comment:
@ambv could you confirm that you're willing to be the sponsor?

Reply:
Assuming he is, please add an entry to CODEOWNERS in the usual form!

A

Reply:
I am.